locales/help_enu/gkhCAIndex.html

```diff
@@ -4,22 +4,22 @@
 <meta charset="utf-8" />
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <link rel="stylesheet" href="styles.css" type="text/css" />
-<title>Authenticity index</title>
+<title>Authenticity Index</title>
 </head>
 <body>
-<h1>Authenticity index</h1>
-<p>Authenticity index of information about person is a special indicator that can be evaluated for person entity only.</p>
-<p>Each reference to a source in a person entity has specific authenticity level of mention that man in the source. Authenticity level can have the following values:</p>
+<h1>Authenticity Index</h1>
+<p>The Authenticity Index is information about a person as a special indicator that can be evaluated for a person entity only.</p>
+<p>Each reference to a source in a person entity has a specific authenticity level of mention for that person in the source. Authenticity level can have the following values:</p>
 <ul>
-<li>Unreliable evidence or estimated data (authenticity is 25%),</li>
-<li>Questionable reliability of evidence (authenticity is 50%),</li>
-<li>Secondary evidence, data officially recorded sometime after event (authenticity is 75%),</li>
+<li>Unreliable evidence or estimated data (authenticity is 25%).</li>
+<li>Questionable reliability of evidence (authenticity is 50%).</li>
+<li>Secondary evidence, data officially recorded sometime after event (authenticity is 75%).</li>
 <li>Direct and primary evidences, or dominance of evidences (authenticity is 100%).</li>
 </ul>
-<p>While you are researching and viewing some information, authenticity level shows to you how much you can trust that information. It is better to set &quot;Questionable reliability of evidence&quot; if a source document had blots, was damaged and could not be read with confidence. This can prevent you from false conclusions.</p>
-<p>If a reference to a source has no authenticity level, such reference gets &quot;Unreliable evidence or estimated data&quot; by default. Just because mentioning a person in document is the important acknowledgement by itself.</p>
-<p>Authenticity index is evaluated automatically for each person basing on authenticity level (as the weighted arithmetic mean). You can view authenticity index
-in <a href="gkhStats.html">statistics</a> module for analysis of mean values or in <a href="gkhTreeChart.html">tree diagrams</a> for pictorial presentation. The index is evaluated using all references to sources, specified
+<p>While you are researching and viewing some information, the authenticity level shows you how much you can trust that information. It is better to set &quot;Questionable reliability of evidence&quot; if a source document had blots, or was damaged and could not be read with confidence. This can prevent you from making false conclusions.</p>
+<p>If a reference to a source has no authenticity level, it gets &quot;Unreliable evidence or estimated data&quot; by default. Mentioning a person in a document is an important acknowledgement by itself.</p>
```

Comment:
The original text contains a logical link between those two sentences.  The second one had no sense without the first one.  Now that link has vanished, and these two sentence are independent from each other.

> Mentioning a person in a document is an important acknowledgement by itself.

This is just a statement of fact, and it doesn't explain why the \`\`Unreliable evidence or estimated data'' is set by default, I believe.

Reply:
Sorry. Here I hope for both of you, because I really don't feel such nuances in the English text.

I can do certain conclusions, focusing on my own text in Russian, but I do not know how best to express a thought in English.

Reply:
This could be something like this: "Because mentioning a person in a document is an important acknowledgement, for a case when a reference to a source has no authenticity level, GEDKeeper uses/assigns \`\`Unreliable evidence or estimated data'' by default".

Reply:
You nailed it!

Reply:
Ack! I did not mean to say this was ready for review. The edit you mentioned must still be made. How do I "unpull" this?

Reply:
Just push new commit in your remote.  I mean, "hide" your previous changes with new ones.  Otherwise, if you would git-rebase(1) and force-pushing to your remote, we may lose† context of this review.

† Because we lose ade04cc commit.

Reply:
**The following is complete off-topic**

> How do I "unpull" this?

For me, reviewing changes after they were published is one of the major github/gitlab disadvantage.  One should be able to review his/her changes **before** one publishes and merges the changes.

Sorry :-(

Reply:
Also off-topic: now that Microsoft owns GitHub perhaps enhancements are forthcoming. Ref: https://www.pcmag.com/feature/361545/6-things-microsoft-plans-to-do-with-github and https://github.blog/2018-10-26-github-and-microsoft/

Reply:
4164c3b7891bf361eeadd6fd48848218416669ae is looked good to me.  👍

What about the previous issue[1]?

[1] https://github.com/Serg-Norseman/GEDKeeper/pull/202#pullrequestreview-231465160

Reply:
Sorry, I've rushed too much with resolving. Text now[1] contains repetitions:

> If a reference to a source has no authenticity level, it gets "Unreliable evidence or estimated data" by default. Because mentioning a person in a document is an important acknowledgement, in cases when reference to a source has no authenticity level, GEDKeeper assigns "Unreliable evidence or estimated data" by default.

I believe we should remove the first sentence.

[1] After 4164c3b

Reply:
What about the following:
If a reference to a source has no authenticity level, it gets "Unreliable evidence or estimated data" by default. Since mentioning a person in a document is an important acknowledgement, in cases when a reference to a source has no authenticity level, GEDKeeper assigns this value.

Reply:
We duplicate "*a reference to a source has no authenticity level*" again.  What if we write something like this:

If a reference to a source has no authenticity level, it gets "Unreliable evidence or estimated data" by default, because mentioning a person in a document is an important acknowledgement.

or:

If a reference to a source has no authenticity level, GEDKeeper assings "Unreliable evidence or estimated data" to that reference by default, because mentioning a person in a document is an important acknowledgement.

Or something else :-)

Reply:
Resolved.  See @Dreamer451's comment[1] below.

[1] https://github.com/Serg-Norseman/GEDKeeper/pull/202#issuecomment-487411757